src/js/App/Sidenav/Navigation/ChromeLink.tsx

```diff
@@ -84,6 +84,9 @@ const LinkWrapper: React.FC<LinkWrapperProps> = ({ href, isBeta, onLinkClick, cl
   };
   const dispatch = useDispatch();
   const onClick = (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => {
+    if (event.ctrlKey) {
```

Comment:
I think we also want to handle a `shift` key click. That opens a new browser window.

Can you also a a test for the `ChromeLink` that the URL wont change after firing an event with ctrt/shift keys pressed?